.github/workflows/ci.yml

```diff
@@ -57,7 +57,64 @@ jobs:
         id: test-action
         uses: ./
         with:
-          pkl-version: 0.25.1
+          pkl-version: 0.26.0
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Confirm download
+        run: pkl --version
+
+  test-action-macos-amd64:
+    name: GitHub Actions Test MacOS amd64
+    runs-on: macos-13
```

Comment:
Let's give it a try - need to see if this is allowed for free with open-source repos. I _think_ it is:

> GitHub Actions usage is free for standard GitHub-hosted runners in public repositories

But it doesn't define what "standard" means (I assume it's "not large runners").

Reply:
looks like it worked 🎉

Reply:
Yep, your definition is correct 🙃.
Nice!